repo/packages/K/kafka-zookeeper/100/resource.json

```diff
@@ -0,0 +1,64 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "netcat": "mesosphere/zk-nc:0.1"
+      }
+    },
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-master-28f8827.tar.gz",
```

Comment:
libmesos-bundle seems to be the old? why not libmesos-bundle-1.11.0.tar.gz?

Reply:
Looks like it hadn't been updated yet.

It's what is presently running, so I'm okay with shipping it like this.